content/advanced/330_servicemesh_using_appmesh/port_to_app_mesh/mesh_resources_overview.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Mesh Resources and Design"
+date: 2020-01-27T08:30:11-07:00
+weight: 10
+---
+
+To port the Product Catalog Apps to App Mesh, you will need create a mesh as well as the various mesh components. You'll also apply labels to the `prodcatalog-ns` namespace to affiliate your new mesh with it, and to enable automatic sidecar injection for pods within it.
+
+#### App Mesh Resources
+
+Kubernetes application objects that run within App Mesh must be defined as `Virtual Nodes`. This provides App Mesh an abstraction to objects such as Kubernetes Deployments and Services, and provides endpoints for communication and routing configuration.
+
+App Mesh also provides the `Virtual Service` construct which allows you to specify a logical service path for application traffic. In this example, they send traffic to `Virtual Routers`, which then route traffic to the `Virtual Nodes`.
+
+In the image below you see Product Catalog App running within App Mesh. Each of the services (`frontend-node`, `prodcatalog`, and `proddetail-v1`) has a virtual node defined, and each application service has a virtual service (`fontend-node`, `prodcatalog` and `proddetail`). These virtual services send traffic to virtual routers within the mesh, which in turn specify routing rules. This drives traffic to their respective virtual nodes and ultimately to the service endpoints within Kubernetes. 
```

Comment:
Can you describe why this is beneficial? e.g. "this enables you to have multiple versions of each service deployed, and you can easily configure traffic routing between them for operations such as blue/green and canary deployments" or similar

Reply:
Updated with details